bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -80,14 +85,27 @@ for idv in idxs:
                 np.divide(errlow2, data2)**2), ys)
     yerr_errhigh = np.multiply(np.sqrt(np.divide(errhigh1, data1)**2 +
                 np.divide(errhigh2, data2)**2), ys)
-    ax_mi.errorbar(xpos + xpos_add_dx * (colorcount - float(len(args.ifars) - 1) / 2.), ys,
+    if args.log_x:
+        xvals = np.exp(np.log(xpos) +
+                       xpos_add_dx * (colorcount -
+                                      float(len(args.ifars) - 1) / 2.))
+    else:
+        xvals = xpos + xpos_add_dx * (colorcount -
+                                      float(len(args.ifars) - 1) / 2.)
+    ax_mi.errorbar(xvals, ys,
         yerr=[yerr_errlow, yerr_errhigh], fmt='o', markersize=7, linewidth=5,
-        label='IFAR = %d yr' % ffirst_set['xvals'][idv], capsize=5, capthick=2, mec='k',
-        color=colors[colorcount % 10])
-    plt.xticks(xpos, labels, rotation='horizontal')
+        label='IFAR = %d yr' % ffirst_set['xvals'][idv], capsize=5,
+        capthick=2, mec='k', color=colors[colorcount % 10])
```

Comment:
I found this was a bug when trying to plot more than 6 IFAR values - since there are 6 colo(u)rs in the list, should be `colorcount % 6`